backend/app/mcp_server/tools/knowledge.py

```
@@ -0,0 +1,492 @@
+
+        db.commit()
+
+        return {
+            "success": True,
+            "document": {
+                "id": document.id,
+                "name": document.name,
+                "source_type": document.source_type,
+                "status": "pending",  # Indexing in progress
+            },
+        }
```

Ruff (0.14.14)

Consider moving this statement to an `else` block

(TRY300)